src/index.js

```diff
@@ -86,7 +86,11 @@ function handlebars(options) {
 
       for (var partial of scanner.partials) {
         // Register the partial dependencies as partials.
-        body += `import '${partial}${options.templateExtension}';\n`;
+        // do not import special partial @partial-block: http://handlebarsjs.com/partials.html#partial-block
```

Comment:
You're right, I should have been post 2 different PRs.

Handlebars is great in hiding best features in doc's :) 
Checkout partial [block with cooperation with inline partial](https://cloudfour.com/thinks/the-hidden-power-of-handlebars-partials/)